pkg/operator/controller/ingress/deployment.go

```diff
@@ -521,22 +521,22 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 	}
 	env = append(env, corev1.EnvVar{Name: RouterHAProxyThreadsEnvName, Value: strconv.Itoa(threads)})
 
-	if ci.Spec.TuningOptions.ClientTimeout != nil && ci.Spec.TuningOptions.ClientTimeout.Duration != 0*time.Second {
+	if ci.Spec.TuningOptions.ClientTimeout != nil && ci.Spec.TuningOptions.ClientTimeout.Duration > 0*time.Second {
```

Comment:
What if Spec.TuningOptions is nil?  Is that possible?

Reply:
> What if Spec.TuningOptions is nil? Is that possible?

It isn't possible.  `Spec.TuningOptions` is not a pointer type.